README.md

```diff
@@ -27,6 +27,7 @@ This repo does not include any assets or assembly code necessary for compiling t
 | ST/NO0/NO0.BIN | ![code coverage NO0.BIN](https://img.shields.io/endpoint?label=NO0%20code&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Fcode%2F%3Fmode%3Dshield%26measure%3Dstno0) | ![decompiled functions](https://img.shields.io/endpoint?label=NO0%20funcs&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Ffunctions%2F%3Fmode%3Dshield%26measure%3Dstno0) | Marble Gallery
 | ST/NO1/NO1.BIN | ![code coverage NO1.BIN](https://img.shields.io/endpoint?label=NO1%20code&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Fcode%2F%3Fmode%3Dshield%26measure%3Dstno1) | ![decompiled functions](https://img.shields.io/endpoint?label=NO1%20funcs&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Ffunctions%2F%3Fmode%3Dshield%26measure%3Dstno1) | Outer Wall
 | ST/NO3/NO3.BIN | ![code coverage NO3.BIN](https://img.shields.io/endpoint?label=NO3%20code&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Fcode%2F%3Fmode%3Dshield%26measure%3Dstno3) | ![decompiled functions](https://img.shields.io/endpoint?label=NO3%20funcs&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Ffunctions%2F%3Fmode%3Dshield%26measure%3Dstno3) | Entrance (first visit)
+| ST/NO4/NO4.BIN | ![code coverage NO4.BIN](https://img.shields.io/endpoint?label=NO4%20code&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Fcode%2F%3Fmode%3Dshield%26measure%3Dstno4) | ![decompiled functions](https://img.shields.io/endpoint?label=NO4%20funcs&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Ffunctions%2F%3Fmode%3Dshield%26measure%3Dstno4) | Underground Caverns
```

Comment:
Sorry I didn't get to reviewing this in time
Are you able to add here as well https://github.com/Xeeynamo/sotn-decomp/blob/master/tools/progress.py#L424
Otherwise progress won't get reported

If not I can raise a PR a bit later

Reply:
All good, I added github.com/Xeeynamo/sotn-decomp/pull/2183